pawno/include/PPC_Common.inc

```diff
@@ -241,14 +241,20 @@ CommandList_Create(playerid)
 }
 
 // This function returns "1" if the given vehicle-id is a plane or helicopter
-IsVehicleAirVehicle(vid)
+IsVehicleAirModel(vid)
```

Comment:
Please change function name back to `IsVehicleBoatVehicle` to follow naming convention.

Reply:
or maybe `IsAirVehicle`...